src/transition.js

```diff
@@ -76,13 +76,13 @@ export function withCurve(transition, curve) {
  * a specified as a map from CSS property names to transition functions for
  * each of these properties.
  * @param {!Element} element
- * @param {!Object<string, !TransitionDef>} styles
- * @return {!TransitionDef<void>}
+ * @param {!Object<string, !TimeTransitionDef>} styles
+ * @return {!TimeTransitionDef<void>}
```

Comment:
@dvoytenko not confident of this change, could you review this. do we call setStyles anywhere where we do pass in TransitionDef instead of TimeTransitionDef (TransitionDef has the type signature (time, complete) and TimeTransitionDef only has (time))